### PR TITLE
Update corporate-identifiers-add.md

### PR DESCRIPTION
**Pull request by msbemba**
Added Android Enterprise corporate-owned devices to the list of automatically assigned corporate-owned status in intune

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/1710

**Comment by msbemba** [comment]
@JohanFreelancer9 please copy review

**Comment by PRMerger8** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/intune/enrollment/corporate-identifiers-add.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ At the time of enrollment, Intune automatically assigns corporate-owned status t
 - Enrolled with a [device enrollment manager](device-enrollment-manager-enroll.md) account (all platforms)
 - Enrolled with the Apple [Device Enrollment Program](device-enrollment-program-enroll-ios.md), [Apple School Manager](apple-school-manager-set-up-ios.md), or [Apple Configurator](apple-configurator-enroll-ios.md) (iOS/iPadOS only)
 - [Identified as corporate-owned before enrollment](#identify-corporate-owned-devices-with-imei-or-serial-number) with an international mobile equipment identifier (IMEI) numbers (all platforms with IMEI numbers) or serial number (iOS/iPadOS and Android)
+- Enrolled with [Android Enterprise corporate-owned devices with work profile](/mem/intune/enrollment/android-corporate-owned-work-profile-enroll).
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/corporate-identifiers-add.md:42]:
```suggestion
- Enrolled with [Android Enterprise corporate-owned devices with work profile](/mem/intune/enrollment/android-corporate-owned-work-profile-enroll)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger8** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@lenewsad Please review and merge this PR. Thanks

**Comment by yogkumgit** [comment]
@lenewsad Gentle reminder. Thanks

**Comment by PRMerger20** [comment]
@lenewsad : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by lenewsad** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**